coverage/PENDING_SUBMIT/amdvlk/0573-COVERAGE-branch-probability-info-574/cov-function-argument-uniform-float-loop-never-return.amber

```diff
@@ -0,0 +1,257 @@
+#!amber
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# A test for a coverage-gap found by the GraphicsFuzz project.
+
+# Short description: A fragment shader that covers specific LLVM code paths
+
+# The test passes because the shader always writes red.
+
+SHADER vertex variant_vertex_shader PASSTHROUGH
+
+# variant_fragment_shader is derived from the following GLSL:
+# #version 320 es
+# #define _int_1 _GLF_uniform_int_values[0]
+# #define _int_0 _GLF_uniform_int_values[1]
+#
+# precision highp float;
+# precision highp int;
+#
+# // Contents of _GLF_uniform_int_values: [1, 0]
+# layout(set = 0, binding = 0) uniform buf0
+# {
+#     int _GLF_uniform_int_values[2];
+# };
+#
+# // Contents of resolution: 256.0
+# layout(push_constant) uniform buf_push
+# {
+#     float resolution;
+# };
+#
+# layout(location = 0) out vec4 _GLF_color;
+#
+# int func(float x)
+# {
+#     int a = 1;
+#
+#     for(int i = 0; i < 2; i ++)
+#     {
+#         while(a < 200)
```

Comment:
Can this be made smaller than 200? I guess not but thought I'd check.

Reply:
Hmm, this one behaves strangely. Most of the lower values makes the coverage to disappear, but I found two values (80 and 110) that do work.